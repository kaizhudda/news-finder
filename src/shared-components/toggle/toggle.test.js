import React from 'react';
import { shallow } from 'enzyme';
import Toggle from './index';

let wrapped = shallow(<Toggle />);

describe('Toggle', () => {
  it('should render the Search Bar Component', () => {
    expect(wrapped).toMatchSnapshot();
  });
});