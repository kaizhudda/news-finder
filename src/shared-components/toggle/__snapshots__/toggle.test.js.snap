// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toggle should render the Search Bar Component 1`] = `
<div
  className="dark-mode-toggle"
>
  <button
    onClick={[Function]}
    type="button"
  >
    ☀
  </button>
  <span
    className="toggle-control"
  >
    <input
      checked={false}
      className="dmcheck"
      id="dmcheck"
      onChange={[Function]}
      type="checkbox"
    />
    <label
      htmlFor="dmcheck"
    />
  </span>
  <button
    onClick={[Function]}
    type="button"
  >
    ☾
  </button>
</div>
`;
